concepticondata/conceptlists/Johansson-2017-56.tsv

```diff
@@ -0,0 +1,57 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Johansson-2017-56-1	1	I	1209	I
+Johansson-2017-56-2	2	YOU	1215	THOU
+Johansson-2017-56-3	3	BIG	1202	BIG
+Johansson-2017-56-4	4	SMALL	1246	SMALL
+Johansson-2017-56-5	5	GOOD	1035	GOOD
+Johansson-2017-56-6	6	BAD	1292	BAD
+Johansson-2017-56-7	7	THIS	1214	THIS
+Johansson-2017-56-8	8	THAT	78	THAT
+Johansson-2017-56-9	9	MUCH/MANY	1198	MANY
+Johansson-2017-56-10	10	FEW	1242	FEW
+Johansson-2017-56-11	11	BEFORE	1882	BEFORE
+Johansson-2017-56-12	12	AFTER	1685	AFTER
+Johansson-2017-56-13	13	ABOVE	1741	ABOVE
+Johansson-2017-56-14	14	BELOW	1485	BELOW OR UNDER
+Johansson-2017-56-15	15	FAR	1406	FAR
+Johansson-2017-56-16	16	NEAR	1942	NEAR
+Johansson-2017-56-17	17	MAN	1554	MAN
+Johansson-2017-56-18	18	WOMAN	962	WOMAN
+Johansson-2017-56-19	19	BLACK	163	BLACK
+Johansson-2017-56-20	20	WHITE	1335	WHITE
+Johansson-2017-56-21	21	HOT	1286	HOT
+Johansson-2017-56-22	22	COLD	1287	COLD
+Johansson-2017-56-23	23	HERE	136	HERE
+Johansson-2017-56-24	24	THERE	1937	THERE
+Johansson-2017-56-25	25	LONG	1203	LONG
+Johansson-2017-56-26	26	SHORT	1645	SHORT
+Johansson-2017-56-27	27	NIGHT	1233	NIGHT
+Johansson-2017-56-28	28	DAY	1225	DAY (NOT NIGHT)
+Johansson-2017-56-29	29	FULL	1429	FULL
+Johansson-2017-56-30	30	EMPTY	1624	EMPTY
+Johansson-2017-56-31	31	NEW	1231	NEW
+Johansson-2017-56-32	32	OLD	1229	OLD
+Johansson-2017-56-33	33	ROUND	1395	ROUND
+Johansson-2017-56-34	34	FLAT	1633	FLAT
+Johansson-2017-56-35	35	DRY	1398	DRY
+Johansson-2017-56-36	36	WET	1726	WET
+Johansson-2017-56-37	37	WIDE/BROAD	2652	BROAD OR WIDE
+Johansson-2017-56-38	38	NARROW	1267	NARROW
+Johansson-2017-56-39	39	THICK	1244	THICK
+Johansson-2017-56-40	40	THIN	2308	THIN
+Johansson-2017-56-41	41	SMOOTH	1234	SMOOTH
+Johansson-2017-56-42	42	ROUGH	1923	ROUGH
+Johansson-2017-56-43	43	HEAVY	1210	HEAVY
+Johansson-2017-56-44	44	LIGHT (not HEAVY)	1052	LIGHT (WEIGHT)
+Johansson-2017-56-45	45	DARK	706	DARK
+Johansson-2017-56-46	46	LIGHT (not DARK)	250	LIGHT (COLOR)
```

Comment:
From the concept descriptions, I'd say that the antonym of dark is 679 BRIGHT
(or the antonym of 250 LIGHT (COLOR) is something like 2633 BLACK OR DARK).

Reply:
According to the author (and luckily he disambiguates this concept), LIGHT (not DARK) means the lack of color so LIGHT (COLOR) fits here.